runtests.sh

```diff
@@ -77,7 +77,7 @@ run_integration(){
         export SNAPCRAFT=snapcraft-coverage
     fi
 
-    python3 -m unittest discover -b -v -s integration_tests
+    SNAPCRAFT_FROM_SOURCE=1 python3 -m unittest discover -b -v -s integration_tests
```

Comment:
ok, I will invert the logic. And please open a bug so I don't forget to add the argument handling to ./runtests integration.